applications/nrf5340_audio/src/audio/sw_codec_select.h

```diff
@@ -69,7 +69,7 @@ enum sw_codec_select_ch {
 	SW_CODEC_STEREO, /**< Use both channels */
 };
 
-enum audio_channel {
+enum audio_channel_select {
```

Comment:
We should probably remove this and use the one in channel_assignment everywhere. I can create a separate JIRA task for it